frontend/micro-ui/web/micro-ui-internals/packages/modules/pgr/src/components/PGRCard.js

```
@@ -36,9 +37,16 @@ const PGRCard = () => {
   useEffect(() => {
     (async () => {
       let response = await Digit.PGRService.count(tenantId, {} );
+      let responseNSLA = await Digit.PGRService.count(tenantId, { 
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'responseNSLA' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)